Formula/commitlint.rb

```diff
@@ -0,0 +1,30 @@
+require "language/node"
+
+class Commitlint < Formula
+  desc "📓 Lint commit messages"
```

Comment:
Not sure if we allow emojis in descriptions. Need another maintainers opinions on this 😄.

Reply:
I agree that should be discussed, please also take a look on formula https://formulae.brew.sh/formula/postgraphile

Reply:
https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/postgraphile.rb

Reply:
Heh. So seems like we allow them, alright.

Reply:
It's not descriptive though, I'd rather have it removed

Reply:
@SMillerDev thx,

how looks like:

`Lint commit messages according to  a commit convention`

Reply:
If that meets the audit requirements it's good to me

Reply:
Exactly, that's the point.

According to the `brew`-convention for new formulas, [commitlint](https://commitlint.js.org/#/) can also be introduced into GitHub Action to check the PRs in advance according to the individual preference, too. 

- https://github.com/conventional-changelog/commitlint

That might be interesting to address in a _new_ Issue.